lib/cloud-watch-event-listener.js

```diff
@@ -168,14 +159,41 @@ class CloudWatchEventListener extends events.EventEmitter {
         metadataSource: 'db',
       };
       
+      // Note that since we're now inserting instances directly into the
+      // database now, we're always going to have a pending state in the db for
+      // managed instances.  This means that most 'pending' state messages will
+      // be ignored.
+      if (dbResponse.lastEvent && dbResponse.lastEvent === generated) {
+        log.warn('whoa!');
```

Comment:
Should we send a warning to sentry instead?

Reply:
It's not strictly an error, not really even something we monitor, more just something I'd be interested to see if it ever happens.